dcos_installer/config_util.py

```diff
@@ -11,19 +11,26 @@
 log = logging.getLogger(__name__)
 
 
-def do_configure(config):
-    gen_out = gen.generate(config.as_gen_format(), extra_sources=[gen.build_deploy.bash.onprem_source])
+def onprem_generate(config):
+    return gen.generate(config.as_gen_format(), extra_sources=[gen.build_deploy.bash.onprem_source])
+
 
+def configure_gen(gen_out):
```

Comment:
The purpose of this function isn't very clear from its name. `make_serve_dir()`?